en/starter/settingup.md

```diff
@@ -24,7 +24,7 @@ redirect_from: "/starter/setting up.html"
 <hr>
 ## Ubuntu Linux
 
-Kitura is tested on Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS.
+Kitura is tested on Ubuntu 14.04 LTS.
```

Comment:
@shmuelk - I was wondering how accurate our documentation is when we tell users that we have tested our code on the following platforms -> Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS. Most of our Swift packages (kitura and non-kitura) use Package-Builder and Travis CI for continuous integration builds and tests. However, Travis CI only supports Ubuntu 14.04. From what I have seen, many of our developers rely on Travis CI to claim that their code works on Linux... which would mean that Ubuntu 14.04 is the only platform that we actually test our packages with. When you say Kitura is tested on Ubuntu 16.04… does that mean that the Kitura team has a continuous integration pipeline for building and testing all of the Kitura packages on Ubuntu 16.04? Or are you saying that developers are manually building and testing their Kitura Swift packages on Ubuntu 16.04? If the latter, I wonder how frequent that manual process occurs (probably less frequent than we think) and how many of our Kitura packages (all of them, or just a few?) are covered by this manual process.